### PR TITLE
feat(post): create react-360 post

### PR DESCRIPTION
**Pull request by nmarini**
Tech Talk - July 29, 2021

### CODE REVIEW

File: src/content/react-360/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+The [react-360](https://github.com/facebookarchive/react-360) framework is for creating interactive 360 experiences that run in your web browser. It pairs modern APIs like WebGL and WebVR with the declarative power of React. React 360 builds off of React Native in its component sturcture and three.js for rendering 3D graphics in the browser.
+
+## Why React 360?
+
+React 360 is a great way for developers new to VR to get an introductory experience. If you have experience with React and React Native then this can be a very effective entry point into VR. React 360 is an great way to introduce some basic concepts to developers new to VR. But, for a more in-depth understanding of VR, a more advanced and robust library would be more useful.
+
+## Install
+
+Before install. you'll need:
```
Comment (trybick) [inline src/content/react-360/index.md]: period maybe should be comma

File: src/content/react-360/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+- Node.js version 6.0.0 of higher
+- `yarn` or `npm` (>=v3.0.0) package manager
+
+Next, install the React 360 CLI – a command-line tool that generates the basic layout of new projects.
+
+```bash
+# npm i -g react-360-cli
```
Comment (trybick) [inline src/content/react-360/index.md]: I think these are coming out as comments, you might not need the `#`

File: src/content/react-360/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+```
+
+PROJECT_NAME represents the name of your new application.
+
+Then you can change directories into your new project and start the application by running `npm start` or `yarn start`.
+When the server has booted you can access your application at `http://localhost:8081/index.html` and you can add `?hotreload` at the end of that url to allow certain changes to the application to automatically update the application without reloading the page.
+
+## Important files/folders:
+
+- `index.js` - Where the main code of you application exists, and contain imports that will determine what your application displays.
```
Comment (trybick) [inline src/content/react-360/index.md]: contains instead of contain maybe

File: src/content/react-360/index.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+- `asset` - In order to reference any resource in your `static_assets` folder you need to use this method.
+- `View` - Allows developers to create a 2D surface and also functions as a `div` container.
+- `Text` - Allows text to be added into 3D space, and is typically nested inside of a `View` component.
+- `Animated` - Makes it possible to toggle the size, position, and appearance of elements in your scene in a declarative style that fits into React.
+- `Entity` - Allows you to display 3D objects in your environment.
+  -- _Note on Entity_-- React 360 does have an `Entity` component but it does not actually work, so most documentation states that you should actually import `Entity` from Entity instead of React 360.
+- `VrButton` - Since the environemnt is represented differently compared to your typical webpage you are not able to interact with the elements on the site like you typically would. React-360 provides the `VrButton` component that is a utility class that detects click-type actions across a variety of input devices.
```
Comment (trybick) [inline src/content/react-360/index.md]: environment spelling

**Review by trybick — COMMENTED** [review]
I was adding a new blog article and noticed your PR was still here, here's a couple quick feedback!

File: src/content/react-360/index.md (hunk 4 above)
Comment (nmarini) [reply src/content/react-360/index.md:63]: > I was adding a new blog article and noticed your PR was still here, here's a couple quick feedback!

Thank you!

**Comment by nmarini** [comment]
> I was adding a new blog article and noticed your PR was still here, here's a couple quick feedback!

Thank you!

**Review by trybick — APPROVED** [review]

**Pull request merged**